File: Basic_Calc.py
# This is a Basic Calculator built in Python 3.7.0
# Coded by Kyle Spencer

from math import *

# Function Definition
def add(num_1, num_2):
    result = num_1 + num_2
    return result

def subtract(num_1, num_2):
    result = num_1 - num_2
    return result

def multiply(num_1, num_2):
    result = num_1 * num_2
    return result

def divide(num_1, num_2):
    result = num_1 / num_2
    return result

# Variable Definition
choice = 0
first_number = 0
second_number = 0
result = 0

# Main Program
print("Welcome to Basic_Calc: Python!")
print("Here are your options:")

print("1. Addition")
print("2. Subtraction")
print("3. Multiplication")
print("4. Division")
print("\n")

choice = input("What would you like to choose?")
choice = float(choice)

if choice == 1:
    print("You have chosen Addition")
    first_number = input("What is your first number? ")
    second_number = input("What is your second number? ")

    result = add(float(first_number), float(second_number))
    print("Your result is " + str(result) + ".")
    print("Thank you for using Basic_Calc: Python")

elif choice == 2:
    print("You have chosen Subtraction")
    first_number = input("What is your first number? ")
    second_number = input("What is your second number? ")

    result = subtract(float(first_number), float(second_number))
    print("Your result is " + str(result) + ".")
    print("Thank you for using Basic_Calc: Python")

elif choice == 3:
    print("You have chosen Multiplication")
    first_number = input("What is your first number? ")
    second_number = input("What is your second number? ")

    result = multiply(float(first_number), float(second_number))
    print("Your result is " + str(result) + ".")
    print("Thank you for using Basic_Calc: Python")

elif choice == 4:
    print("You have chosen Division")
    first_number = input("What is your first number? ")
    second_number = input("What is your second number? ")

    result = divide(float(first_number), float(second_number))
    print("Your result is " + str(result) + ".")
    print("Thank you for using Basic_Calc: Python")

elif choice == 69:
    print("Nice!")

else:
    print("You have entered an invalid selection.")
    print("Thank you for using Basic_Calc: Python")

File: Basic_Calc.java
// This is a Basic Calculator built in Java
// Coded by Kyle Spencer

import java.util.Scanner;

public class Basic_Calc {
    public static void main(String[] args) {
        // Variable Declaration
        int selection_int;
        float first_number;
        float second_number;
        float result;
        Scanner input = new Scanner(System.in);

        // Main Program Body
        System.out.println("Welcome to Basic Calculator: Java!");
        System.out.println("\n");
        
        System.out.println("Here are your options:");
        System.out.println("1: Addition");
        System.out.println("2: Subtraction");
        System.out.println("3: Multiplication");
        System.out.println("4: Division");
        System.out.println("\n");

        System.out.println("What is your selection?");
        String selection = input.next();
        selection_int = Integer.parseInt(selection);

        if (selection_int == 1) {
            System.out.println("You have chosen Addition!");
            System.out.println("Please enter your first number.");
            String number_one = input.next();
            first_number = Float.parseFloat(number_one);

            System.out.println("Please enter your second number.");
            String number_two = input.next();
            second_number = Float.parseFloat(number_two);

            result = first_number + second_number;
            System.out.println(result);
        }
        else if (selection_int == 2) {
            System.out.println("You have chosen Subtraction!");
            System.out.println("Please enter your first number.");
            String number_one = input.next();
            first_number = Float.parseFloat(number_one);

            System.out.println("Please enter your second number.");
            String number_two = input.next();
            second_number = Float.parseFloat(number_two);

            result = first_number - second_number;
            System.out.println(result);
        }
        else if (selection_int == 3) {
            System.out.println("You have chosen Multiplication!");
            System.out.println("Please enter your first number.");
            String number_one = input.next();
            first_number = Float.parseFloat(number_one);

            System.out.println("Please enter your second number.");
            String number_two = input.next();
            second_number = Float.parseFloat(number_two);

            result = first_number * second_number;
            System.out.println(result);
        }
        else if (selection_int == 4) {
            System.out.println("You have chosen Division!");
            System.out.println("Please enter your first number.");
            String number_one = input.next();
            first_number = Float.parseFloat(number_one);

            System.out.println("Please enter your second number.");
            String number_two = input.next();
            second_number = Float.parseFloat(number_two);

            result = first_number / second_number;
            System.out.println(result);
        }
        else if (selection_int == 69) {
            System.out.println("Nice!");
        }
        else {
            System.out.println("You have entered an invalid selection.");
            System.out.println("Thank you for using Basic_Calc: Java");
        }

        input.close();
    }
}